### PR TITLE
fix: run create-advisory late in the rh-advisories pipeline

### PR DESCRIPTION
**Pull request by mmalina**
Task `create-advisory` would run after `publish-pyxis-repository` which would ensure it runs late in the pipeline. But with the changes in 1.5.0, `publish-pyxis-repository` runs much earlier now.

So make `create-advisory` run after `push-rpm-data-to-pyxis` which is what `publish-pyxis-repository` ran after originally.

### CODE REVIEW

**Review by happybhati — DISMISSED** [review]

File: pipelines/rh-advisories/rh-advisories.yaml
Comment (johnbieren) [inline pipelines/rh-advisories/rh-advisories.yaml:553]: Can you add run-file-updates and rh-sign-image so it is truly last (before the finally and update-cr-status)? I think that is all the additional ones needed

**Review by johnbieren — CHANGES_REQUESTED** [review]

File: pipelines/rh-advisories/rh-advisories.yaml
Comment (mmalina) [reply pipelines/rh-advisories/rh-advisories.yaml:553]: OK, what about rh-sign-image-cosign? It's not really used yet, but probably makes sense? Hopefully when that one's skipped, it won't block this.

Comment (mmalina) [reply pipelines/rh-advisories/rh-advisories.yaml:553]: I added `rh-sign-image-cosign` as well. Let me know if you disagree.

Comment (johnbieren) [reply pipelines/rh-advisories/rh-advisories.yaml:553]: > OK, what about rh-sign-image-cosign? It's not really used yet, but probably makes sense? Hopefully when that one's skipped, it won't block this.

I don't know, but that should certainly be tested before we merge this

Comment (mmalina) [reply pipelines/rh-advisories/rh-advisories.yaml:553]: The e2e test does create an advisory and it definitely does not run rh-sign-image-cosign, so it should cover this, no?

Comment (johnbieren) [reply pipelines/rh-advisories/rh-advisories.yaml:553]: No idea if the rh-sign-image-cosign task is run in e2e. My personal faith in e2e is kind of low lately as well. But you may be right. It would also be easy to test in a temp unit test file. I just don't want to merge it until we know for sure that task A having runAfter a skipped task still executes task A

File: pipelines/rh-advisories/README.md
Comment (johnbieren) [inline pipelines/rh-advisories/README.md:26]: great entry

**Review by johnbieren — APPROVED** [review]
Approving pending Scott's testing reflects what we expect to happen

File: pipelines/rh-advisories/rh-advisories.yaml
Comment (mmalina) [reply pipelines/rh-advisories/rh-advisories.yaml:553]: Scott will test this today.

Comment (scoheb) [reply pipelines/rh-advisories/rh-advisories.yaml:553]: work of art:

adv-perfect

**Comment by mmalina** [comment]
> Approving pending Scott's testing reflects what we expect to happen

Scott's test was successful. He did both runs - one with rh-sign-image-cosign and one when it was skipped. Both worked.

**Pull request merged**
